iis-wf/iis-wf-export-actionmanager/pom.xml

```diff
@@ -39,6 +39,12 @@
             <version>${project.version}</version>
         </dependency>
 
+        <dependency>
+            <groupId>eu.dnetlib.dhp</groupId>
+            <artifactId>dhp-common</artifactId>
+            <version>1.2.4-SNAPSHOT</version>
```

Comment:
We need to make sure to remove this SNAPSHOT dependency and everything SNAPSHOT related.